thunder/tests/test_networks.py

```diff
@@ -419,10 +419,6 @@ def test_thunderfx_mistral_nemo_small():
     assert mdl._backend.subgraph_infos, "Should have at least 1 subgraph"
 
 
-@pytest.mark.skipif(
```

Comment:
This change seems unrelated to the rest of the PR - maybe it should be in a separate PR?

Reply:
Right, it should be merged separately. It slipped through as I was testing whether HF models work.